pyproject.toml

```diff
@@ -174,3 +175,31 @@ markers =[
     "unit",
 ]
 log_level = "info"
+filterwarnings = [
+    #"error",  # Turn all warnings not explicitly filtered below into errors
```

Comment:
@Shinnnyshinshin you forgot to uncomment this line.
When I do I still see the same errors.